lnrpc/switchrpc/switch.proto

```diff
@@ -15,6 +15,12 @@ service Switch {
     the full details of the payment route.
```

Comment:
I think it would be nice to include parts of the next commit message from `switchrpc: add TrackOnion rpc` here to describe how to use the rpc regarding state ambiguity.